specification/confidentialledger/Microsoft.mCCF/recovery.tsp

```diff
@@ -0,0 +1,59 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+
+import "./common.tsp";
+
+@TypeSpec.Versioning.useDependency(Azure.Core.Versions.v1_0_Preview_2)
+namespace Recovery;
+
+@doc("A share of a recovery key, granted for a specific recovery member, and encrypted with that member's share-encryption key. This is safe to share in the ledger or amongst untrusted callers, as only the intended member will be able to decrypt and access the secret content.")
+@TypeSpec.Rest.resource("encrypted-recovery-shares")
+model EncryptedRecoveryShare {
+  @doc("ID of CCF member who this recovery share belongs to.")
+  @visibility("read")
+  @key
+  memberId: memberId;
+}
+
+@doc("Container for a member's raw recovery share. This should be carefully guarded, and only submitted to a trusted service over a secure channel, for the purposes of recovery.")
+model RecoveryShare {
+  @doc("Base-64 encoding of a member's raw recovery share (unencrypted).")
+  share: string;
```

Comment:
I don't recall *exactly* what this in TypeSpec, but better to declare as `bytes` with a base64 encoding so the generators do the right thing here by default.

Reply:
I've now updated all of these base64-encoded values to bytes, which matches what we're passing around internally so should produce closer generated code. There's some knock-on issues with providing realistic example values (https://github.com/Azure/oav/pull/979), but they're downstream and we can ignore for now if necessary.